debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/Module.java

```diff
@@ -35,4 +35,11 @@ public static String name() {
     public static String contextName() {
         return "Postgres";
     }
+
+    /**
+     * @return logical name of the connector used in mbean object name
+     */
+    public static String logicalName() {
```

Comment:
`logicalName`  sounds very generic. The comment says it is used as MBean object name. Is it still right to use that generic name here? @jpechane

Reply:
I think the method is unnecessary and duplicates `contextName()`. If lowercase is needed for querying then just call `Module.contextName().toLowercase()`

Reply:
@jpechane That can done for `postgres`. But `contextName()` for SQL Server doesn't match. It should be `sql_server`.

Reply:
@ani-sha It can, `sql_server` is just lower case variant of `SQL_Server`

Reply:
@ani-sha I would prefer addding a "metricsName()` method and put the same strings there as you did here.
```suggestion
    public static String metricsName() {
```
and leave the rest unchanged. I am not a fan of doing the lowercase call. It would be executed on every call to `#metricsName()`.

Reply:
We agreed on that we will use `contextName().toLowerCase()` for now and I gonna file a Jira to replace all occurences of `contextName().toLowerCase()` with an additional parameter passed to the `Metrics` implementation in core with the respective name in order that with a single PR / task all metrics base on the same code.